lib/apollo-federation/tracing/tracer.rb

```diff
@@ -116,16 +116,18 @@ def self.execute_field(data, &block)
 
         end_time_nanos = Process.clock_gettime(Process::CLOCK_MONOTONIC, :nanosecond)
 
-        # interpreter runtime
+        # legacy runtime
         if data.include?(:context)
           path = context.path
           field_name = context.field.graphql_name
           field_type = context.field.type.to_s
           parent_type = context.parent_type.graphql_name
-        else # legacy runtime
+        else # interpreter runtime
           path = data.fetch(:path)
-          field_name = data.fetch(:field).graphql_name
-          field_type = data.fetch(:field).type.unwrap.graphql_name
+          field = data.fetch(:field)
+          field_name = field.graphql_name
+          field_type = field.type.unwrap.graphql_name
```

Comment:
Good call to write a spec @grxy . This has always been broken by the test caught it. The `graphql_name` of the type doesn't include whether it's nullable, hence the need to check the field